packages/stylelint-plugin/tsconfig.dist.json

```diff
@@ -9,7 +9,8 @@
     "declarationMap": true,
     "declaration": true,
     "target": "es5",
-    "module": "CommonJS"
+    "module": "CommonJS",
+    "noUncheckedIndexedAccess": true
```

Comment:
This (and the other packages) shouldn't require it since they all extend the root dist file

Reply:
ahhh I knew you did some magic recently - I'll update now

Reply:
I'll remove from design-tokens too as that was previously added before this update.

Reply:
So did you want me to put it back? @HeartSquared

Reply:
Hmm... it will probably be better for the releases since I suspect it would release an unnecessary patch. But I'm not too fussed either.

Reply:
@HeartSquared Reverted and ready for another tick :)